doc/src/build/move.md

```diff
@@ -387,7 +387,13 @@ name = "MyFirstPackage"
 version = "0.0.1"
 
 [dependencies]
-Sui = { local = "https://github.com/MystenLabs/sui/tree/main/sui_programmability/framework/" }
```

Comment:
this does not work for me

Reply:
Please describe the error you encounter.

Reply:
Figured this out. It should be:

```
Sui = { local = "../sui_programmability/framework/" }

```

With a relative path. Fixing in a separate PR. Thank you for finding this!